client.go

```diff
@@ -92,19 +97,20 @@ func (c *Castle) Risk(ctx context.Context, req *Request) (RecommendedAction, err
 	if req.Context == nil {
 		return RecommendedActionNone, errors.New("request.Context cannot be nil")
 	}
-	r := &castleAPIRequest{
+	r := &castleRiskAPIRequest{
 		Type:         req.Event.EventType,
 		Name:         req.Event.Name,
 		Status:       req.Event.EventStatus,
 		RequestToken: req.Context.RequestToken,
 		User:         req.User,
 		Context:      req.Context,
 		Properties:   req.Properties,
+		CreatedAt:    req.CreatedAt,
```

Comment:
Possibly worth defaulting to `time.Now()` if the created at isn't set in the request, that will preserve existing behaviour and avoid a scenario where we update to latest `castle-go` without updating the Risk calls to set created at?